lib/ret/login.ex

```diff
@@ -22,4 +24,33 @@ defmodule Ret.Login do
     |> unique_constraint(:identifier_hash)
     |> put_assoc(:account, account)
   end
+
+  def update_identifier_hash(%{old_email: old_email, new_email: new_email} = _) do
```

Comment:
This isn’t the right module for this function.

Additionally the `= _` is needless.

Reply:
Where does it belong? I don't have a good sense of code organization in elixir apps

Reply:
A useful pattern for managing state is: functional core, imperative shell. Struct manipulation is low-level, which make stateful calls (e.g., `Repo` calls) within a struct module a design smell.

How to fix? Reserve this module for pure functions that operate on `%Ret.Login{}` and put any new repo calls in a module outside this one (e.g., a context boundary module).

Reply:
Resolved in https://github.com/mozilla/reticulum/pull/636/commits/eee56865456bf34ae00958478c1ed8e15a49007b